### ISSUE TITLE
Problem in ctrls_builtin demo

### ISSUE THREAD

**Issue by mrezai** [bruvzg/godot_tl#4]
OS: ubuntu 18.04
Godot: 3.1.1-stable
godot_tl: master

After compile godot with godot_tl module, import ctrls_builtin demo and Run it from in project manager(before open it in godot editor), text will be rendered correctly:
![Screenshot from 2019-10-25 22-12-19](https://user-images.githubusercontent.com/5120436/67624794-5b038c80-f842-11e9-9e9f-f1610bfb824f.png)

After open it in godot editor and run:
![Screenshot from 2019-10-25 22-13-56](https://user-images.githubusercontent.com/5120436/67624814-b170cb00-f842-11e9-9ff0-15b27151c4f3.png)

Close editor and open it again:
![Screenshot from 2019-10-25 22-21-49](https://user-images.githubusercontent.com/5120436/67624824-cc433f80-f842-11e9-9699-13d7c826e828.png)

It seems there is a problem in importing project to editor because font properties in Control.tscn file  changed like this diff:

```diff
diff --git a/demos/ctrls_builtin/Control.tscn b/demos/ctrls_builtin/Control.tscn
index 3853fe4..52cde41 100644
--- a/demos/ctrls_builtin/Control.tscn
+++ b/demos/ctrls_builtin/Control.tscn
@@ -1,25 +1,17 @@
-[gd_scene load_steps=9 format=2]
+[gd_scene load_steps=7 format=2]
 
 [ext_resource path="res://fonts/NotoSans-Bold.ttf" type="DynamicFontData" id=1]
 [ext_resource path="res://fonts/NotoNastaliqUrdu-Regular.ttf" type="DynamicFontData" id=2]
 
-[sub_resource type="TLDynamicFontFace" id=1]
-font_path = "res://fonts/NotoSans-Bold.ttf"
+[sub_resource type="TLFontFamily" id=1]
 
-[sub_resource type="TLFontFamily" id=2]
-regular/0 = SubResource( 1 )
-
-[sub_resource type="DynamicFont" id=3]
+[sub_resource type="DynamicFont" id=2]
 size = 12
 font_data = ExtResource( 1 )
 
-[sub_resource type="TLDynamicFontFace" id=4]
-font_path = "res://fonts/NotoNastaliqUrdu-Regular.ttf"
-
-[sub_resource type="TLFontFamily" id=5]
-regular/0 = SubResource( 4 )
+[sub_resource type="TLFontFamily" id=3]
 
-[sub_resource type="DynamicFont" id=6]
+[sub_resource type="DynamicFont" id=4]
 size = 12
 font_data = ExtResource( 2 )
 
@@ -34,14 +26,14 @@ margin_right = 321.0
 margin_bottom = 146.0
 text = "test232
 gdtl TLLabel"
-base_font = SubResource( 2 )
+base_font = SubResource( 1 )
 
 [node name="Label" type="Label" parent="."]
 margin_left = 401.0
 margin_top = 37.0
 margin_right = 611.0
 margin_bottom = 151.0
-custom_fonts/font = SubResource( 3 )
+custom_fonts/font = SubResource( 2 )
 text = "test232
 Godot Label"
 
@@ -50,7 +42,7 @@ margin_left = 100.0
 margin_top = 159.0
 margin_right = 322.0
 margin_bottom = 200.0
-base_font = SubResource( 5 )
+base_font = SubResource( 3 )
 text = "اور بازار سے لے آئے اگر ٹوٹ گیا"
 
 [node name="LineEdit" type="LineEdit" parent="."]
@@ -58,5 +50,5 @@ margin_left = 390.0
 margin_top = 159.0
 margin_right = 612.0
 margin_bottom = 200.0
-custom_fonts/font = SubResource( 6 )
+custom_fonts/font = SubResource( 4 )
 text = "اور بازار سے لے آئے اگر ٹوٹ گیا"
```

**Comment by bruvzg** [comment]
It's regression from bc7d9c2c290660ddd7cc1302ad79e39ee9a22be1 and 2453f74026e24d11921485555f9c1786646ec975 changes.

Should be fixed by #5, please check it out.

**Comment by mrezai** [comment]
:+1: 
PR #5 fixed the issue but I found another issue accidentally!
Select TLLineEdit and click on "Font Path" (res://fonts/...) then click on scene in editor, the result is(render problem in TLLineEdit's text):
![Screenshot from 2019-10-27 00-38-46](https://user-images.githubusercontent.com/5120436/67626066-4976b080-f853-11e9-864f-ee959311beff.png)

and lots of error in console:
```
ERROR: getornull: Condition ' !id_map.has(p_rid.get_data()) ' is true. returned: __null
   At: ./core/rid.h:164.
ERROR: getornull: Condition ' !id_map.has(p_rid.get_data()) ' is true. returned: __null
   At: ./core/rid.h:164.
ERROR: getornull: Condition ' !id_map.has(p_rid.get_data()) ' is true. returned: __null
   At: ./core/rid.h:164.
```

**Comment by bruvzg** [comment]
> Select TLLineEdit and click on "Font Path" (res://fonts/...) then click on scene in editor, the result is(render problem in TLLineEdit's text)

It's caused by cache not updating automatically on font change, should be fixed by #6.

**Comment by mrezai** [comment]
:+1: 
PR #6 Fixed it.
Thanks
I will test other demos and report if I find something
